docs/home/ARM.md

```diff
@@ -0,0 +1,568 @@
+# ARM
+
+From Epiphany v1.1.0 preliminary support for the ```arm64``` architecture was added. As the ```arm64``` architecture is relatively new to the datacenter at the time of writing only a subset of providers, operating system, components and applications are supported. Support will be extended in the future when there is a need for it.
+
+## Support
+
+Below we give the current state of ```arm64``` support across the different providers, operating systems, components and applications. Make sure to check the ***notes*** for limitations that might still be present for supported components or applications.
+
+Besides making sure that the selected providers, operating systems, components and applications are supported with the tables below any other configuration for ```Epicli``` will work the same on ```arm64``` as they do on ```x86_64```. ```Epicli``` will return an error if any configuration is used that is not supported by the ```arm64``` architecture.
+
+### Providers
+
+| Providers | CentOS 7.x | RedHat 7.x | Ubuntu 18.04 |
```

Comment:
I would rather use singular (Provider, Component, Application) in all tables, or at least keep them consistent